iphone/Maps/Classes/CustomAlert/Toast/Toast.swift

```diff
@@ -9,10 +9,21 @@ final class Toast: NSObject {
   private var blurView = UIVisualEffectView(effect: UIBlurEffect(style: .dark))
   private var timer: Timer?
 
+  private static var toasts: [Toast] = []
+
   @objc static func toast(withText text: String) -> Toast {
-    return Toast(text)
+    let toast = Toast(text)
+    toasts.append(toast)
+    return toast
   }
-
+  
+  @objc static func hideAll(){
+    var toastsCopy = toasts
+    toastsCopy.forEach {
+      $0.hide()
+    }
```

Comment:
nit: it's better to write in single line easy iterations `toastsCopy.forEach { $0.hide() }`